docs/getting-started.qmd

```diff
@@ -66,40 +71,63 @@ Please see below how to configure the spftware to connect to your iRODS instance
 
 ### iBridges config.json
 
-*iBridges* will create its own configuration file in `~/.ibridges/` containing the name of the last *iRODS* environment file used.  This `config.json` file can be updated to control other aspects of *iBridges*.  For example:
+*iBridges* will create its own configuration file in `~/.ibridges/ibridges_config.json` containing the name of the last *iRODS* environment file used.  This `ibridges_config.json` file can be updated to control other aspects of *iBridges*.  For example:
 
 ```json
 {
     "last_ienv": "irods_environment.json",
     "davrods_server": "https://server.fqdn.nl",
+    "eln_token": "<api key>",
+    "amber_token": "<api key>"
     "ui_tabs": [
```

Comment:
In a recent PR I have added the ability to configure the order of the tabs in the main menu simply by using that order here.  `tabBrowser` and `tabInfo` are added automatically to the beginning and end, respectively, if they are not included here.  A user can put them where they want if they do include them here.

It would be a good idea to add them here explaining at least the existing functionality.  I can then update it in that PR.